rfcs/20180827-api-names.md

```diff
@@ -0,0 +1,587 @@
+| tf.assert_integer | tf.debugging.assert_integer |
+| tf.assert_less | tf.debugging.assert_less |
+| tf.assert_less_equal | tf.debugging.assert_less_equal |
+| tf.assert_near | tf.debugging.assert_near |
+| tf.assert_negative | tf.debugging.assert_negative |
+| tf.assert_non_negative | tf.debugging.assert_non_negative |
+| tf.assert_non_positive | tf.debugging.assert_non_positive |
+| tf.assert_none_equal | tf.debugging.assert_none_equal |
+| tf.assert_positive | tf.debugging.assert_positive |
+| tf.assert_proper_iterable | tf.debugging.assert_proper_iterable |
+| tf.assert_rank | tf.debugging.assert_rank |
+| tf.assert_rank_at_least | tf.debugging.assert_rank_at_least |
+| tf.assert_rank_in | tf.debugging.assert_rank_in |
+| tf.assert_same_float_dtype | tf.debugging.assert_same_float_dtype |
+| tf.assert_scalar | tf.debugging.assert_scalar |
+| tf.assert_type | tf.debugging.assert_type |
+| tf.bfloat16 | tf.dtypes.bfloat16 |
+| tf.bincount | tf.math.bincount |
+| tf.bool | tf.dtypes.bool |
+| tf.boolean_mask | tf.manip.boolean_mask |
+| tf.cast | tf.dtypes.cast |
+| tf.complex | tf.dtypes.complex |
+| tf.complex128 | tf.dtypes.complex128 |
+| tf.complex64 | tf.dtypes.complex64 |
+| tf.concat | tf.manip.concat |
+| tf.confusion_matrix | tf.train.confusion_matrix |
+| tf.conj | tf.math.conj |
+| tf.count_nonzero | tf.math.count_nonzero |
+| tf.cumprod | tf.math.cumprod |
+| tf.cumsum | tf.math.cumsum |
+| tf.decode_csv | tf.io.decode_csv |
+| tf.depth_to_space | tf.nn.depth_to_space, tf.manip.depth_to_space |
+| tf.deserialize_many_sparse | tf.io.deserialize_many_sparse |
+| tf.divide | tf.math.divide |
+| tf.double | tf.dtypes.double |
+| tf.erf | tf.math.erf |
+| tf.expand_dims | tf.manip.expand_dims |
+| tf.float16 | tf.dtypes.float16 |
+| tf.float32 | tf.dtypes.float32 |
+| tf.float64 | tf.dtypes.float64 |
+| tf.floordiv | tf.math.floordiv |
+| tf.floormod | tf.math.floormod |
+| tf.gather | tf.manip.gather |
+| tf.get_seed | tf.random.get_seed |
+| tf.global_norm | tf.losses.global_norm |
+| tf.half | tf.dtypes.half |
+| tf.imag | tf.math.imag |
+| tf.import_graph_def | tf.graph_util.import_graph_def |
+| tf.int16 | tf.dtypes.int16 |
+| tf.int32 | tf.dtypes.int32 |
+| tf.int64 | tf.dtypes.int64 |
+| tf.int8 | tf.dtypes.int8 |
+| tf.is_non_decreasing | tf.debugging.is_non_decreasing |
+| tf.is_numeric_tensor | tf.debugging.is_numeric_tensor |
+| tf.is_strictly_increasing | tf.debugging.is_strictly_increasing |
+| tf.lbeta | tf.math.lbeta |
+| tf.log_sigmoid | tf.math.log_sigmoid |
+| tf.logical_xor | tf.math.logical_xor |
+| tf.matmul | tf.linalg.matmul |
+| tf.mod | tf.math.mod |
+| tf.multinomial | tf.random.multinomial |
+| tf.multiply | tf.math.multiply |
+| tf.negative | tf.math.negative |
+| tf.newaxis | tf.manip.newaxis |
+| tf.nn.in_top_k | tf.math.in_top_k |
+| tf.nn.l2_normalize | tf.math.l2_normalize, tf.linalg.l2_normalize |
+| tf.nn.log_poisson_loss | tf.losses.log_poisson_loss |
```

Comment:
`log_poisson_losss` feels like a slightly strange fit for `tf.losses`, given that it's a low-level function that doesn't adhere to the tf.losses interface (e.g., with the `loss_collection` and `reduction` arguments).